### ISSUE TITLE
non-ASCII characters in json files

### ISSUE THREAD

**Issue by ojnas** [Telecominfraproject/oopt-gnpy#139]
Several of the json topology files in the example folder contains "\u2192" in the fiber element uids, which looks strange. I believe this is because they were converted from xls and `gnpy/core/convert.py` uses the unicode character "→" but json.dumps, which is used to convert to json, outputs only ASCII by default. If you want to include non-ASCII in the json, you can change the writing to file as follows:

`edfa_json_file.write(dumps(data, ensure_ascii=False, indent=2))`

But maybe the safer option would be to not use non-ASCII characters and instead replace "→" with e.g. "->".

**Comment by dutc** [comment]
Since these files may contain both "analytical" and "business" data, we need to be able to deal with non-ASCII content.

We can assume that gnpy .json files are encoded in UTF-8 as per https://tools.ietf.org/html/rfc8259

We should document this somewhere.

I think we can leave this as-is or toggle `ensure_ascii` when we write these out. This will only affect the ease of hand-editing the generated .json files. Irrespective of whether we toggle this option, these will be round-tripped correctly:

```python
>>> from json import dumps, loads
>>> loads(dumps('→'))
'→'
```

**Comment by dutc** [comment]
Should be fixed with #142
